eng/pipelines/templates/steps/analyze.yml

```diff
@@ -5,31 +5,7 @@ parameters:
   AdditionalTestArgs: ''
 
 steps:
-  - task: UsePythonVersion@0
-    displayName: 'Use Python $(PythonVersion)'
-    inputs:
-     versionSpec: '$(PythonVersion)'
-
-  - task: DownloadPipelineArtifact@0
-    inputs:
-      artifactName: 'artifacts'
-      targetPath: $(Build.ArtifactStagingDirectory)
-
-  - script: |
-     pip install -r eng/ci_tools.txt
-     ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/eng/.docsettings.yml
-    displayName: 'Verify Readmes'
-
-  - pwsh: |
-      mkdir "$(Build.ArtifactStagingDirectory)/reports"
-      Copy-Item -Path "$(Build.SourcesDirectory)/eng/common/InterdependencyGraph.html" -Destination "$(Build.ArtifactStagingDirectory)/reports/InterdependencyGraph.html"
-    displayName: 'Populate Reports Staging Folder'
-
-  - task: PythonScript@0
-    displayName: 'Analyze dependencies'
-    inputs:
-     scriptPath: 'scripts/analyze_deps.py'
-     arguments: '--verbose --out "$(Build.ArtifactStagingDirectory)/reports/dependencies.html" --dump "$(Build.ArtifactStagingDirectory)/reports/data.js"'
+  - template: /eng/pipelines/templates/steps/analyze_dependency.yml
```

Comment:
I suspect you need to pass some of the parameters to the template through to this template. Parameters don't automatically flow.

Reply:
I will recheck this again. Here is a successful run of aggregate reports using current change.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=472866&view=results

Reply:
I have removed usused parameters in analyze-dependency.yml